src/ILCompiler.Compiler/src/Compiler/DependencyAnalysis/EETypeNode.cs

```diff
@@ -144,6 +144,17 @@ protected override DependencyList ComputeNonRelocationBasedDependencies(NodeFact
             return null;
         }
 
+        public override bool InterestingForDynamicDependencyAnalysis
```

Comment:
Based on your offline explanation to me of what a ConstructedEETypeNode is used for, yes this should go there :)

---
In reply to: [96538247](https://github.com/dotnet/corert/pull/2521#discussion_r96538247) [](ancestors = 96538247)